### PR TITLE
[trival] fix typo

### PR DESCRIPTION
**Pull request by CaoShuFeng**
reopen https://github.com/kubernetes/kubernetes/pull/47300

**Release note**:

```
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/plugin/pkg/audit/doc.go [hunk 1]
```diff
@@ -14,5 +14,5 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package auduút contains implementations for pkg/audit/AuditBackend interface
+// Package auduit contains implementations for pkg/audit/AuditBackend interface
```
Comment (wojtek-t) [inline staging/src/k8s.io/apiserver/plugin/pkg/audit/doc.go]: Still incorrect. Should be "audit".

Comment (CaoShuFeng) [inline staging/src/k8s.io/apiserver/plugin/pkg/audit/doc.go]: Done.

**Comment by CaoShuFeng** [comment]
Fixed hack/verify-golint.sh

**Pull request merged**